src/Components/ResourcePicker/ResourcePicker.tsx

```diff
@@ -113,12 +114,14 @@ const ResourcePicker: React.FC<IResourcePickerProps> = ({
     });
 
     const loadingLabelText = useMemo(() => {
-        switch (resourceType) {
-            case AzureResourceTypes.DigitalTwinInstance:
+        switch (
+            resourceType.toLowerCase() // need to compare lowercase since for the resource types that we use Resource Graph api, it returns lowercased resource type in the response object
+        ) {
+            case AzureResourceTypes.DigitalTwinInstance.toLowerCase():
                 return t('resourcesPicker.loadingInstances');
-            case AzureResourceTypes.StorageBlobContainer:
+            case AzureResourceTypes.StorageBlobContainer.toLowerCase():
```

Comment:
is there a reason we don't just lowercase the enum value itself?

Reply:
yes, I kept the enum as is since **other management service provider endpoint call return the enums as it was** (including upper case) but since now the resource graph api returns all lowercase, I needed to add these lowercase comparison because we don't know the return object based on the api call in management adapter. Since Resource Graph allows querying only [certain types](https://learn.microsoft.com/en-us/azure/governance/resource-graph/reference/supported-tables-resources#resources), we still use service provider calls for containers for example where the types include upper case for initial letters.